base/client.jl

```diff
@@ -19,12 +19,24 @@ const text_colors = AnyDict(
 )
 
 have_color = false
+default_color_warn_key = :red
+default_color_info_key = :blue
 @unix_only default_color_answer = text_colors[:bold]
 @unix_only default_color_input = text_colors[:bold]
 @windows_only default_color_answer = text_colors[:normal]
 @windows_only default_color_input = text_colors[:normal]
 color_normal = text_colors[:normal]
 
+function warn_color_key()
```

Comment:
This works exactly like the existing `answer_color` mechanism – the point of checking every time is so that you can change it during a running REPL session. That's a bit fancy, but it would be weird to have one of these work like that and not the others.